properties/P000146.md

```diff
@@ -0,0 +1,10 @@
+---
+uid: P000146
+slug: ultraparacompact
+name: Ultraparacompact
+refs:
+  - mathse: 4673639
+    name: Answer to What separation properites are satisfied by the Arens space?
```

Comment:
For ultraparacompact I think a more focused ref would be the arxiv paper you mentioned, which has a doi:
https://doi.org/10.48550/arXiv.1306.6086